server/endpoint/health/searcher/endpoint.go

```diff
@@ -77,23 +79,39 @@ func (e *Endpoint) Encoder() kithttp.EncodeResponseFunc {
 
 func (e *Endpoint) Endpoint() kitendpoint.Endpoint {
 	return func(ctx context.Context, request interface{}) (interface{}, error) {
-		var serviceRequest searcher.Request
-		{
-			clusterID, ok := request.(string)
-			if !ok {
-				return nil, microerror.Mask(badRequestError)
-			}
-			serviceRequest = searcher.Request{
-				ClusterID: clusterID,
-			}
+		clusterID, ok := request.(string)
+		if !ok {
+			return nil, microerror.Mask(badRequestError)
 		}
 
-		serviceResponse, err := e.service.Health.Searcher.Search(ctx, serviceRequest)
+		clusterRequest := cluster.Request{
+			ClusterID: clusterID,
+		}
+		clusterResponse, err := e.service.Cluster.Searcher.Search(ctx, clusterRequest)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+
+		nodeRequest := node.Request{
+			ClusterID: clusterID,
+			Endpoint:  clusterResponse.Endpoint,
+		}
+		nodeResponse, err := e.service.Node.Searcher.Search(ctx, nodeRequest)
```

Comment:
In the same way `Node` does not seem very obvious - this is collection of information from the _tenant_ cluster which currently is only nodes.

Reply:
I renamed this to `Tenant`.